code/utility/src/palindrome/palindrome_check/Readme.md

```diff
@@ -0,0 +1,28 @@
+# Palindrome Checker
+
+The aim is to find if the string is palindrome or not. The string is taken as input from the user.
+
+## Explaination 
+
+- String= "agaga" : Palindrome = YES #The string agaga is same when read from forward or backward. Hence it is a palindrome
+- String= "abccba" : Palindrome = YES
+- String= "rotor" : Palindrome = YES
+- String= "abbaab" : Palindrome = NO
+
+## Algorithm
+
+Palindrome is defined as a string which reads the same backwards as forwards. If there are odd number of characters in the string, we ignore the middle character and check for palindrome. 
```

Comment:
A palindrome is defined ... backwards _and_ forwards. If there are _an_ odd number ..., and check whether the remaining string is a palindrome.